README.md

```diff
@@ -1,11 +1,10 @@
-# Kafka Airflow Provider
-
-
-> **Warning**
-
-> This package has been deprecated after being accepted to OSS Airflow. Please use apache-airflow[apache.kafka] instead if you're looking for a supported kafka provider. 
+## ⚠️ Discontinuation of project
+> This project is no longer actively maintained by Astronomer, but we’d love to see it live on in the community. While Astronomer has paused development and is not accepting new contributions, bug fixes or releases, the code is still here for you to explore, fork and adapt under the terms of its license. 
+> Please note that it may not work with the latest dependencies or platforms, and it could contain security vulnerabilities. Astronomer can’t offer guarantees or warranties for its use.
```

Comment:
```suggestion
> Please note that the code available in [this repository](https://github.com/astronomer/airflow-provider-kafka) may not work with the latest dependencies or platforms, and it could contain security vulnerabilities. Astronomer can’t offer guarantees or warranties for its use.
```